docs/route-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+Consider the specifications of a Worker:
+- A Worker is meant to handle only the Tasks that it is registered to handle. This means that Workers registered to handle Tasks for Workflow A are not capable of handling Tasks for Workflow B, and vice versa. 
```

Comment:
```suggestion
- A Worker only handles the Tasks that it is registered to handle. This means that Workers registered to handle Tasks for Workflow A are not capable of handling Tasks for Workflow B, and vice versa. 
```

Other than rewording the first sentence, I think the description is a bit confusing. It sort of suggests that a Worker can't handle tasks for both Workflow A and B inherently. Obviously I can have Worker listen on a single taskqueue and support multiple types (A and B).